Frameworks/Plugins/ZappGoogleAnalytics/Files/Universal/GoogleAnalyticsPluginAdapter.swift

```diff
@@ -138,27 +137,35 @@ import ZappCore
 }
 
 extension GoogleAnalyticsPluginAdapter: AnalyticsProviderProtocol {
+    public func disable(completion: ((Bool) -> Void)?) {
+        isDisabled = true
+    }
+
     public var providerName: String {
-        return "Google Analytics"
+        return "GoogleAnalytics"
     }
 
-    public func prepareProvider(_ mandatoryDefaultParams: [String: Any],
-                                completion: (_ isReady: Bool) -> Void) {
+    public func prepareProvider(_ defaultParams: [String: Any],
```

Comment:
I suggest to move the methods related to PluginAdapterProtocol to the main class where the PluginAdapterProtocol is added and have here just the methods related to GoogleAnalyticsPluginAdapter, 
what do you think?